mobile_payment/gql_mutations.py

```diff
@@ -189,163 +179,166 @@ def async_mutate(cls, user, **data):
         except Exception as exc:
-            logger.exception("transaction.mutation.failed_to_create_transaction")
-            return [{
-                    'message': _("transaction.mutation.failed_to_create_transaction"),
-                    'detail': str(exc)}] 
+            logger.exception("transaction.mutation.failed_to_create_initiate_transaction")
+            return InitiateTransactionMutation(
+                responseMessage=_("transaction.mutation.failed_to_create_initiate_transaction"), 
+                detail=str(exc), 
+                Success=False
+            )
         
-    
-        
-class  ProcessTransactionMutation(OpenIMISMutation):
+class ProcessTransactionMutation(graphene.relay.ClientIDMutation):
     """
-    create a mutaion for making a trasaction
+    create a mutation to update a payment transaction
     """
-
     _mutation_module = "mobile_payment"
     _mutation_class = "ProcessTransactionMutation"
-
-    class Input(TransactionBase, OpenIMISMutation.Input):
+    
+    class Input(PaymentTransactionInputType):
         pass
-
+    
+    responseMessage = graphene.String()
+    responseCode = graphene.String()
+    Success = graphene.Boolean()
+    detail = graphene.String()
+    
     @classmethod
-    def mutate_and_get_payload(cls, root, info, **data) -> Optional[str]:
-        response = super().mutate_and_get_payload(root, info, **data)
+    def mutate_and_get_payload(cls, root, info, **data):
         try:
             if info.context.user is AnonymousUser:
                 raise ValidationError(_("mutation.authentication_required"))
-            if not info.context.user.has_perms(MobilePaymentConfig.gql_mutation_create_transaction_perms):
+            if not info.context.user.has_perms(MobilepaymentConfig.gql_mutation_create_payment_transaction_perms):
                 raise PermissionDenied(_("unathorized"))
             client_mutation_id = data.get("client_mutation_id")
             transaction_uuid = data['uuid']
-            transacton = Transactions.objects.get(uuid=transaction_uuid )
+            transacton = PaymentTransaction.objects.get(uuid=transaction_uuid)
             if not transacton:
-                raise PermissionDenied (_("invalid transaction_uuid"))
-            transaction_id = transacton.transaction_id
+                raise PermissionDenied(_("invalid transaction_uuid"))
+            transaction_id = transacton.psp_transaction_id
             otp = data['otp']
-            print(f' Transaction_id = {transaction_id}')
-            print (f'otp =  {otp}')
-            Qcell_process = process_request(otp, transaction_id)
-            if Qcell_process['responseCode'] == '1':
+            logger.info(f'Transaction_id = {transaction_id}')
+            logger.info(f'otp = {otp}')
+            qmoney_process = process_request(otp, transaction_id)
+            if qmoney_process['responseCode'] == '1':
```

Comment:
I've found the doc by myself.

In general, I expect the doc of a third party service (It's fine to share a link if it's public), especially when it's not easy to get access (such as the Django framework for instance). This will help the future maintenance, especially with magic constant and their meaning.